webrtc.html

```diff
@@ -3445,62 +3445,68 @@
-                <ul>
+                in <var>description</var> then perform the following checks:</p>
+                <ol>
                   <li>
-                    <p>If <var>t</var>'s <a>[[\Direction]]</a> slot is
+                    <p>If <var>transceiver</var>'s <a>[[\Direction]]</a> slot is
                     <code>"sendrecv"</code> or <code>"sendonly"</code>,
-                    and the <a>associated</a> m= section in
-                    <var>connection</var>'s <code><a data-for=
-                    "RTCPeerConnection">currentLocalDescription</a></code>
+                    and the <a>associated</a> m= section in <var>description</var>
                     doesn't contain an "a=msid" line, return <code>true</code>.
                     </p>
                   </li>
                   <li>
-                    <p>If <var>connection</var>'s <code><a data-for=
-                    "RTCPeerConnection">currentLocalDescription</a></code> is of
-                    type <code>"offer"</code>, and the direction of the <a>associated</a> m=
-                    section in neither the offer nor answer matches
-                    <var>t</var>'s <a>[[\Direction]]</a>
+                    <p>If <var>description</var> is of type <code>"offer"</code>,
+                    and the direction of the <a>associated</a> m=
+                    section in neither <var>connection</var>'s
+                    <code><a data-for=
+                    "RTCPeerConnection">currentLocalDescription</a></code> nor
+                    <code><a data-for=
+                    "RTCPeerConnection">currentRemoteDescription</a></code>
+                    matches <var>transceiver</var>'s <a>[[\Direction]]</a>
                     slot, return <code>true</code>.</p>
                   </li>
                   <li>
-                    <p>If <var>connection</var>'s <code><a data-for=
-                    "RTCPeerConnection">currentLocalDescription</a></code> is of
-                    type <code>"answer"</code>, and the direction of the <a>associated</a> m=
-                    section in the answer does not match <var>t</var>'s
+                    <p>If <var>description</var> is of type <code>"answer"</code>,
+                    and the direction of the <a>associated</a> m=
+                    section in the answer does not match <var>transceiver</var>'s
                     <a>[[\Direction]]</a> slot intersected with
                     the offered direction (as described in <span data-jsep=
                     "initial-answers">[[!JSEP]]</span>), return
                     <code>true</code>.</p>
                   </li>
-                </ul>
+                </ol>
               </li>
               <li>
-                <p>If <var>t</var> is <a data-for="RTCRtpTransceiver">
+                <p>If <var>transceiver</var> is <a data-for="RTCRtpTransceiver">
                 stopped</a> and is <a>associated</a> with an m= section, but the
```

Comment:
@taylor-b I leave the session description here unspecified, because this seem to suggest it can be associated with any of the 4 session descriptions.